spring-boot/autoconfigure/src/main/java/org/togglz/spring/boot/test/TogglzTestExecutionListener.java

```diff
@@ -1,3 +1,19 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 package org.togglz.spring.boot.test;
```

Comment:
Should this also go into the `autoconfigure` package? I basically like the idea of having one unique package per module. Especially because duplicated packages between different modules will break with Java 9 modules.

Reply:
I'm not sure about this one...  Honestly I think this is not related with auto-configuration, and I think that we probably should move this into another module, for instance, `togglz-spring-core`.

Reply:
Ah, I see. This class actually doesn't use any Spring Boot classes, correct? In this case `togglz-spring-core` would be a better place.